### DISCUSSION TITLE
Trying To Integrate Mock Service Worker and Not Sure Where To Start

### DISCUSSION THREAD

**Discussion by aaronksaunders** [nuxt/nuxt#16225] [Questions]
Want to be able to do some basic development without running my database or even just work disconnected and I thought integrating MSW would be a great approach.

the first attempt was trying to load a server plugin to set it up, but not sure my plugin is even getting called.

any pointers would be appreciated

**Comment by declaresub** [comment]
I think I have Mock Service Worker playing with nuxt.  I am currently using nuxt-bridge. I  hope but don't really expect that all should just work in nuxt 3.  If you're still interested, let me know and I will see about posting some information.

**Reply by maxarias-io** [reply]
@declaresub This is pretty high in the search results, mind posting the gist of your solution here?

**Reply by declaresub** [reply]
Here is a gist containing my configuration.

https://gist.github.com/declaresub/9a7b8b4f5f57e21bf05003fdf64788aa

So far, my configuration also works with nuxt-proxy; any resource for which there is no MSW request handler is proxied as one would expect.
